### ISSUE TITLE
[Feature Request]: Custom Recipes in Beepedia for Pack Devs

### ISSUE THREAD

**Issue by TemperedFool** [Resourceful-Bees/ResourcefulBees#270]
### Is your feature request related to a problem?

Lots of confused players not finding the (admittedly silly and confusing) recipes I've made for making bees.

### Solution(s)

I would like to be able to register recipes to show up in the Beepedia. Best would be if I can also specify where it goes - is this for making the bee (so probably under breeding), is it using the bee, using honeycombs, etc. A catch-all "recipes related to this bee" would work.

Being completely ignorant of how impossible to implement this will be, my idea was something like this: allow a list of recipe ID's in a section of the bee json, probably with subsections for each "page" of the beepedia. Bonus points if we can do this via datapack/Kube, but that is likely a bridge too far.

### Describe alternatives you've considered

What I may do instead is just extend the 50 shades of bees book; but having everything consolidated into the beepedia seems preferable.

### Mod Version

0.6.7.2b

### Mod Loader Version

1.16.5-some_forge_version

### Additional context

Sorry, on mobile and having trouble retrieving the current version of Forge that E6 is using - I can get it if it's relevant.

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Vaerys-Dawn** [comment]
already considering adding a "obtain_text" field to the breedData section, so that if you fill that out with text it'll show up in the breed tab along side the other breeding recipes

**Comment by TemperedFool** [comment]
Yeah, that will get the job done, but I'm imagining something similar to Patchouli's recipe pages. It's a big ask for sure ^_^
